RELEASE.md

```diff
@@ -0,0 +1,37 @@
+# Release Process
+
+Pack follows a 6 week release cadence, composed of 3 phases:
+  - [Development](#development)
+  - [Feature Complete](#feature-complete)
+  - [Release Finalization](#release-finalization)
+
+## Development
+Our development flow is detailed in [Development](DEVELOPMENT.md)
+
+## Feature Complete
+### Process
+5 business days prior to a scheduled release, we enter `feature complete`. A release branch (in the form `release/<VERSION>`) is created, and used for `UAT` (User Acceptance testing).
```

Comment:
```suggestion
5 business days prior to a scheduled release, we enter `feature complete`. A release branch (in the form `release/<VERSION>`) is created, and used for User Acceptance testing (`UAT`).
```

Reply:
Resolved in becdf38e75f60d7a9553381dfd4b8f9bb928c21f